### PR TITLE
Small changes to service overrides guide

### PR DESCRIPTION
**Pull request by dussault-antoine**
### Merge instructions

- [ ] Please merge after reviewing

### CODE REVIEW

File: content/en/tracing/guide/service_overrides.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ Inferred services improve how Datadog represents service dependencies. This docu
 
 ### Before inferred services
 
-Datadog changed service names of client spans (`span.kind:client`) to represent databases, queues, and third-party dependencies. For example, a client call from service `A` to a PostgreSQL database would be tagged as `service:postgres` or `service:A-postgres`. 
+Datadog used to change service names of client spans (`span.kind:client`) to represent databases, queues, and third-party dependencies. For example, a client call from service `A` to a PostgreSQL database would be tagged as `service:postgres` or `service:A-postgres`. Changing the service name of spans will be referred to as a [**service override**](#service-override) in the rest of this guide. The initial service name will be referred to as the [**base service**](#base-service)
```
Comment (urseberry) [inline content/en/tracing/guide/service_overrides.md:21]:
```suggestion
Datadog used to change service names of client spans (`span.kind:client`) to represent databases, queues, and third-party dependencies. For example, a client call from service `A` to a PostgreSQL database would be tagged as `service:postgres` or `service:A-postgres`. Changing the service name of spans is referred to as a [**service override**](#service-override) in the rest of this guide. The initial service name is referred to as the [**base service**](#base-service).
```

File: content/en/tracing/guide/service_overrides.md [hunk 2]
```diff
@@ -98,11 +98,11 @@ For example:
 - .NET tags gRPC calls as `service:<DD_SERVICE>-grpc-client`
 - Python tags gRPC calls as `service:grpc-client`
 
-With the `DD_TRACE_REMOVE_INTEGRATION_SERVICE_NAMES_ENABLED` option set to `true`, all supported tracing libraries tag client spans capturing the call to the downstream service with the calling service's name, `service:<DD_SERVICE>`. This provides a *global default service name*.
+With the `DD_TRACE_REMOVE_INTEGRATION_SERVICE_NAMES_ENABLED` option set to `true`, all supported tracing libraries tag client spans capturing the call to the downstream service with the calling service's name, `service:<DD_SERVICE>`. This ensure all spans are always tagged with the *default service name* emitting the span; [`peer.*`][6] attributes are used to describe the called dependency (database, queue, etc...).
```
Comment (urseberry) [inline content/en/tracing/guide/service_overrides.md:101]:
```suggestion
With the `DD_TRACE_REMOVE_INTEGRATION_SERVICE_NAMES_ENABLED` option set to `true`, all supported tracing libraries tag client spans capturing the call to the downstream service with the calling service's name, `service:<DD_SERVICE>`. This ensures all spans are always tagged with the *default service name* emitting the span; [`peer.*`][6] attributes are used to describe the called dependency (for example, database or queue).
```

File: content/en/tracing/guide/service_overrides.md [hunk 3]
```diff
@@ -133,6 +133,14 @@ Similarly, for a span representing a call to a mySQL database:
 
 **Note**: The configuration above only removes [integration service overrides](#integration-service-overrides). Custom service overrides must be removed directly in the code.
 
+## Glossary
```
Comment (urseberry) [inline content/en/tracing/guide/service_overrides.md:136]: Having a small glossary for this page alone limits its use. Can you add these terms to the main glossary instead? You can see it here: https://docs.datadoghq.com/glossary/.

Here are instructions for adding a term to the glossary: https://datadoghq.atlassian.net/wiki/spaces/docs4docs/pages/3154742084/Glossary#Creating-a-glossary-entry. If you don't have the documentation build set up locally, ask for help in Slack in #documentation.

**Review by urseberry — CHANGES_REQUESTED** [review]
Thank you for adding to the public docs!

File: content/en/tracing/guide/service_overrides.md (hunk 3 above)
Comment (dussault-antoine) [reply content/en/tracing/guide/service_overrides.md:136]: I'd like to keep this guide isolated ideally, as these terms are not used anywhere else for now. Once we move this page to the main docs, I'll move these terms to the global glossary

**Review by urseberry — APPROVED** [review]

**Pull request merged**